config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -380,6 +380,42 @@ presubmits:
           privileged: true # TODO(fejta): https://github.com/kubernetes/kubernetes/issues/76484
 
   kubernetes/perf-tests:
+  - name: pull-perf-tests-benchmark-kube-dns
+    always_run: false
+    skip_report: false
+    max_concurrency: 3
+    branches:
+      - master
+    annotations:
+      testgrid-dashboards: sig-scalability-benchmarks
```

Comment:
Good catch with the dashboards. Let's change this as well, it should be in the presubmit dashboard:
```
testgrid-dashboards: presubmits-kubernetes-scalability
```

Reply:
done.